src/components/Options/style.scss

```diff
@@ -54,4 +54,5 @@ $horizontalPadding: 15px;
 .options-scroller {
   overflow-x: hidden;
   overflow-y: auto;
+  -webkit-overflow-scrolling: touch;
```

Comment:
Android does something with this? 😮

Reply:
Ah, no sorry, I snuck that in as an iOS fix.